vendor.mod

```diff
@@ -4,7 +4,9 @@
 
 module github.com/docker/docker
 
-go 1.20
+go 1.21
+
+toolchain go1.22.5
```

Comment:
Unintentional change when vendoring. Reverting this piece.

Reply:
Ah, yes, ran into that myself a few times